Services/CO.CDP.OrganisationInformation.Persistence.Tests/DatabasePersonTenantLookupTest.cs

```diff
@@ -135,7 +138,8 @@ public async Task ItFindsThePersonWithAllTheirTenants()
             {
                 Email = acmeCoPersonWithNoOrganisationConnection.Email,
                 Name = $"{acmeCoPersonWithNoOrganisationConnection.FirstName} {acmeCoPersonWithNoOrganisationConnection.LastName}",
-                Urn = acmeCoWithNoOrganisationPersonUrn
+                Urn = acmeCoWithNoOrganisationPersonUrn,
+                Scopes = []
```

Comment:
Can you make sure one of these tests actually verifies non empty scopes are returned from a person?

Reply:
done.